phoenix-core/src/it/java/org/apache/phoenix/end2end/index/PartialIndexRebuilderIT.java

```diff
@@ -1075,6 +1111,12 @@ public void testIndexFailureWithinRSDisablesIndex() throws Throwable {
     public static class WriteFailingRegionObserver extends SimpleRegionObserver {
         @Override
         public void postBatchMutate(ObserverContext<RegionCoprocessorEnvironment> c, MiniBatchOperationInProgress<Mutation> miniBatchOp) throws IOException {
+            // we need to advance the clock, since the index retry logic (copied from HBase) has a time component
+            EnvironmentEdge delegate = EnvironmentEdgeManager.getDelegate();
+            if (delegate instanceof MyClock) {
+                MyClock myClock = (MyClock) delegate;
+                myClock.time += 1000;
+            }
```

Comment:
Do we still need this since you're using TableRef.getCurrentTime()? If yes, that's ok.